AlgoStoreData.cs/DTOs/InstanceDataDTO.cs

```diff
@@ -464,7 +467,7 @@ public static InstanceDataDTO returnInstanceDataDTO(string AlgoId)
                                     new AlgoMetaDataParameter
                                     {
                                         Key = "StartingDate",
-                                        Value = "10-02-2018",
+                                        Value = "01-04-2018",
```

Comment:
Is it a good idea to make this date as parameter?

Reply:
Yes. This will class will be refactored in another PR.